packages/client/src/TRPCClientError.ts

```diff
@@ -1,11 +1,10 @@
-import { DefaultErrorShape } from '@trpc/server';
-import { TRPCErrorResponse } from '@trpc/server/rpc';
+import { DefaultErrorShape, Maybe } from '@trpc/core';
 import {
   getCauseFromUnknown,
   inferErrorShape,
   TRPCInferrable,
-} from '@trpc/server/shared';
-import { Maybe } from '@trpc/server/unstableInternalsExport';
+} from '@trpc/core/shared';
```

Comment:
i don't like the name `shared` for an entrypoint coming from a `core` package, where the whole point is to share code 🤷‍♂️